### ISSUE TITLE
training loss 0.0 learning rate 0.0

### ISSUE THREAD

**Issue by Nanamumuhan** [runwayIA/alpaca-lora#1]
May I ask if you have a loss of 0 and a learning rate of 0 during your training.
![image](https://github.com/runwayIA/alpaca-lora/assets/41236260/8917f398-e741-49d4-8afa-13f1f150d81e)

**Comment by Nanamumuhan** [comment]
![image](https://github.com/runwayIA/alpaca-lora/assets/41236260/63b6811b-45e4-4f2e-9f15-82296c119ec1)
![image](https://github.com/runwayIA/alpaca-lora/assets/41236260/c72b256c-6fee-4c6d-8bf5-2782e0573b05)
